docs/data/elasticsearch/transaction.json

```diff
@@ -115,6 +115,7 @@
         },
         "name": "GET /api/types",
         "result": "success",
+        "sampled": true,
```

Comment:
please remove from the `minimal` example as it is not a required field.

Reply:
I don't think this is a `minimal` example?